locale/en.po

```diff
@@ -2275,10 +2275,14 @@ msgstr "HP:"
 msgid "HSV"
 msgstr "HSV"
 
+#: ../src/microbe_stage/gui/HUDMessages.cs:37
+msgid "HUD_MESSAGE_MULTIPLE"
+msgstr "{0} (x {1})"
```

Comment:
A bit strange to have a space after the x, isn't this usually written as (x2), (x10), and so on? We already have `SPECIES_N_TIMES` too.

Reply:
Yeah, that makes sense. When quickly writing out all the English text I didn't even notice this.